### PR TITLE
Fixes #108 add authn ldap support

### LINKED ISSUE #108: Support authn-ldap conjur logon

### PR DESCRIPTION
**Pull request by infamousjoeg**

### CODE REVIEW

File: cmd/conjur.go [hunk 1]
```diff
@@ -396,6 +404,7 @@ func init() {
 	conjurLogonCmd.MarkFlagRequired("account")
 	conjurLogonCmd.Flags().StringVarP(&BaseURL, "base-url", "b", "", "Conjur appliance URL")
 	conjurLogonCmd.MarkFlagRequired("base-url")
+	conjurLogonCmd.Flags().StringVarP(&AuthnLDAP, "authn-ldap", "", "", "Uses provided Service ID to configure LDAP Authentication")
```
Comment (AndrewCopeland) [inline cmd/conjur.go:407]: Uses should be User?

File: docs/cybr_conjur_logon.md [hunk 2]
```diff
@@ -16,11 +16,12 @@ cybr conjur logon [flags]
 ### Options
 
 ```
-  -a, --account string    Conjur account
-  -b, --base-url string   Conjur appliance URL
-  -h, --help              help for logon
-  -l, --login string      Conjur login name
-      --self-signed       Retrieve and use self-signed certificate when sending requests to the Conjur API
+  -a, --account string      Conjur account
+      --authn-ldap string   Uses provided Service ID to configure LDAP Authentication
```
Comment (AndrewCopeland) [inline docs/cybr_conjur_logon.md:20]: Same here

File: pkg/cybr/conjur/logon.go [hunk 3]
```diff
@@ -44,7 +44,7 @@ func Login(applianceURL string, account string, username string, password []byte
 		return nil, err
 	}
 
-	url := fmt.Sprintf("%s/authn/%s/login", applianceURL, url.QueryEscape(account))
+	url := fmt.Sprintf("%s/%s/login", applianceURL, url.QueryEscape(account))
```
Comment (AndrewCopeland) [inline pkg/cybr/conjur/logon.go:47]: `applianceURL` should be changed to `authnURL`

**Review by AndrewCopeland — CHANGES_REQUESTED** [review]
Take a look at my comments :D

File: cmd/conjur.go (hunk 1 above)
Comment (infamousjoeg) [reply cmd/conjur.go:407]: No, I meant for `Uses` like describing it's action. Do you think it's more coherent to make it `User`?

File: docs/cybr_conjur_logon.md (hunk 2 above)
Comment (infamousjoeg) [reply docs/cybr_conjur_logon.md:20]: Pending your answer to my question in the previous comment.

File: pkg/cybr/conjur/logon.go (hunk 3 above)
Comment (infamousjoeg) [reply pkg/cybr/conjur/logon.go:47]: Is that just for readability?  (Just wondering...)

Comment (AndrewCopeland) [reply pkg/cybr/conjur/logon.go:47]: Yes, authnURL shows that it is a `authnURL` and it is not a `applianceURL`

**Pull request merged**